### ISSUE TITLE
zip'd model needed

### ISSUE THREAD

**Issue by KoichiYasuoka** [clarinsi/classla-stanfordnlp#3]
`classla.download("sl")` downloads five files, whose sizes are 2.1GB total. I've tried to `zip` the five files and got 1.4GB. To speed up the download, would you kindly prepare zip'd model?

**Comment by nljubesi** [comment]
Thank you for your suggestion. The thing is that we retrieve the models for each language from a repository (https://www.clarin.si/repository/xmlui/) where the models are properly stored (with metadata, handle etc.) and these are not zipped on that side. We will consider submitting to the repository maximally compressed files in the future.
@TomazErjavec @lkrsnik

**Comment by TomazErjavec** [comment]
> We will consider submitting to the repository maximally compressed files in the future.

I wasn't aware that these binary files can be sensible compressed. But, if so, let me know which handles should have their files compressed, and I will do it. You will then have to change the upload procedure of course, to retrieve zips and unzip them.

**Comment by KoichiYasuoka** [comment]
First of all, among the five files, [ssj500k.pretrain.pt](http://hdl.handle.net/11356/1312) is the biggest one to download for one hour (from Japan). I'm very eager to `zip` it 70% (or `xz` it 50%). The second one is [ssj500k+Sloleks_lemmatizer.pt](https://www.clarin.si/repository/xmlui/handle/11356/1286) and the other three files are relatively small.

**Comment by nljubesi** [comment]
@KoichiYasuoka Thank you once again for your suggestion. After considering multiple scenarios, we decided to add support for zipped models and to compress newly published models (such as http://hdl.handle.net/11356/1348 and http://hdl.handle.net/11356/1349).

We decided not to compress old models as this might end up in code breakage for some users. Given that we are publishing new models rather frequently (new lemmatization models to be published this week), most users will profit from your suggestions quite soon.
